packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3835,17 +3884,26 @@ class ErrorWidget extends LeafRenderObjectWidget {
   /// corresponds to a [RenderBox] that can handle the most absurd of incoming
   /// constraints. The default constructor maps to a [RenderErrorBox].
   ///
+  /// The default behavior is to show the exception's message in debug mode,
+  /// and to show a short message in release builds.
+  ///
   /// See also:
   ///
   ///  * [FlutterError.onError], which is typically called with the same
   ///    [FlutterErrorDetails] object immediately prior to this callback being
   ///    invoked, and which can also be configured to control how errors are
   ///    reported.
-  static ErrorWidgetBuilder builder = (FlutterErrorDetails details) => ErrorWidget(details.exception);
+  static ErrorWidgetBuilder builder = _defaultErrorWidgetBuilder;
 
-  /// The message to display.
-  final String message;
-  final FlutterError _flutterError;
+  static Widget _defaultErrorWidgetBuilder(FlutterErrorDetails details) {
+    String message = '[sic]'; // could also consider "‽"
```

Comment:
Why not just `[error]` to make it clear that this is not supposed to be here, but something went wrong. Or something a little more more friendly like "Oh snap, something went wrong here" (borrowing from Chrome).

If I see [sic] in an app I've no idea what that's supposed to be....

Reply:
Yeah, I wasn't sure what to put here.

I don't want to put "error" or something like that because the whole point is to make this friendly.

I don't want to put ":-(" or an emoji or "Oh snap" or similar, because that would leak our personality into Flutter apps, and the whole point is to not do that.

Maybe it should just not show any text at all?

Reply:
If that's the goal I wouldn't put any text at all (that would also get around the problem that any string we put here really needs to be localized).

However, I also find that odd from a usability perspective. As a user I'd much rather know that something is broken over seeing a state that looks kinda OK but may not work as expected. The latter is far more frustrating. It makes me think that we may just want to crash...

Reply:
Crashing isn't really reasonable when the ErrorWidget only appears for a tiny 10x10 part of the screen for one frame of a long animation, though...

Reply:
removed "[sic]"